scripts/word.py

```diff
@@ -0,0 +1,58 @@
+"""
+Interacting with MS Word
+------------------------
+
+One use case in reproducible science that works great in the context of
+LaTeX files is the separation of text and images, where the images are
+generated automatically in separate workflows. For MS Word, the challenge
+is to update images of a document automatically.
+
+A MS Word document is basically a zip file containing a text layer and a 
+picture layer. Those pictures are, if unzipped, stored in word/media and can 
+be accessed via
+
+"""
+import zipfile
+
+z = zipfile.ZipFile("report.docx")
+all_files = z.namelist()
+images = [f for f in all_files if f.startswith("word/media/")]
+print(images)
+
+"""
+You can now modify the contents (actually creating a new zip file) to replace 
+one of the images with another one. This is conveniently done by the provided
+``class UpdatableZipFile`` (web search: *python replace file in zip* lead to 
+https://stackoverflow.com/a/35435548 )
+
+You can direcly overwrite the *report.docx*, but we chose to work on a copy of
+it (*report_modified.docx*) here.
+
+"""
+
+from upgradable_zip_file import *
```

Comment:
What are you importing other than `UpdateableZipFile`? Also, why is the class *updateable*, and the file *upgradeable*?